src/commands/go-test.yaml

```diff
@@ -19,6 +19,8 @@ steps:
       echo -n " -X '<< parameters.pkg >>.version=$(cat .project_version)'" >> .ldflags
   - run: |
       echo "\"$(cat .ldflags)\""
+  - run: |
+      go get golang.org/x/tools/cmd/goimports && find . -path ./vendor -prune -o -name '*.go' -exec goimports -local github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} -d "{}" +
```

Comment:
Projects using this orb are using go modules. So I guess we don't have to do the find magic.

But still I'd prefer to make this somehow fail if goimports thinks imports are not fine.

Reply:
Make it fail 💯

Reply:
:fire: